src/harness/reference_models/pre_iap_filtering/fss_purge.py

```diff
@@ -0,0 +1,296 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+    This is a subset of the Pre-IAP reference model which implements
+    the FSS purge list algorithm described in R2-SGN-29.
+    The main function is fssPurgeModel().
+==================================================================================
+"""
+import numpy as np
+from reference_models.antenna import antenna
+from reference_models.interference import interference
+from reference_models.geo import vincenty
+from reference_models.propagation import wf_itm
+import pre_iap_util
+# Definition of constants and data types for FSS purge model.
+# Reference Channel bandwidth for FSS OOBE calculation
+REF_BW = 5.e6
+
+# OOBE Margin
+FSS_OOBE_MARGIN = 1
+
+def getFssInfo(fss_record):
+  """ Create a FssProtectionPoint tuple from the fss_record.
+  Args:
+    fss_record: A single FSS record dictionary.
+  Returns:
+    FssProtectionPoint: Tuple holding details about FSS needed for interference calculation.
+  """
+
+  # Extract fields from FSS record. 
+  fss_latitude = fss_record['record']['deploymentParam'][0]['installationParam']['latitude']
+  fss_longitude = fss_record['record']['deploymentParam'][0]['installationParam']['longitude']
+  fss_height = fss_record['record']['deploymentParam'][0]['installationParam']['height']
+  fss_height_type = fss_record['record']['deploymentParam'][0]['installationParam']['heightType']
+  fss_azimuth = fss_record['record']['deploymentParam'][0]['installationParam']['antennaAzimuth']
+  fss_pointing_elevation = fss_record['record']['deploymentParam'][0]['installationParam']['elevation']
```

Comment:
Should be extracted using the antennaDowntilt key and not the elevation

Reply:
Modified   key as "antennaDowntilt" for fss_pointing_elevation calculation